Containers/recording/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3.11.3-alpine3.18
+
+COPY --chmod=775 start.sh /usr/bin/start.sh
+COPY --chmod=664 recording.conf /etc/recording.conf
+
+RUN set -ex; \
+    apk add --no-cache \
+        ca-certificates \
+        tzdata \
+        bash \
+        ffmpeg \
+        libpulse \
+        bind-tools \
+        netcat-openbsd \
+        git \
+        wget \
+        shadow; \
+    # xvfb firefox chromium chromium-chromedriver?
+    # apk add --no-cache geckodriver --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing; \
+    useradd --system recordning; \
+    git clone --recursive https://github.com/nextcloud/spreed --branch v16.0.3 /src; \
```

Comment:
```suggestion
    git clone https://github.com/nextcloud/spreed --branch v16.0.3  --depth 1 /src; \
```

Reply:
and why? maybe there are at some point submodules needed

Reply:
true, then just lets add --depth 1

Reply:
what does this do?